_data/other.yml

```diff
@@ -343,6 +343,13 @@ websites:
       tfa: Yes
       software: Yes
 
+    - name: Ulule
+      url: https://www.ulule.com/
+      twitter: ulule
+      facebook: ulule
```

Comment:
Oh yes i just see.
Yes the **ulule** page is the French Page (cause created in France before expand for EU)
So yes, better for here to put **ululeEn** page